js_src/app.js

```diff
@@ -17,13 +17,14 @@
  * Copyright (C) 2011 Simon Newton
  */
 
+goog.provide('app.setup');
+
+goog.require('app.displayCommand');
```

Comment:
I was going to say you don't need this, but I'm guessing it optimises it out if it's not in there otherwise?

Reply:
If it's not in there, I get a `TypeError: app.displayCommand is not a function` in the browser console on PID pages. However, I just noticed another error: `TypeError: this.l.ya is not a function` 😕

Reply:
Hmm, then you have to look at the minification to make sense of it! Or perhaps try the non-advanced optimisation may be enough to tell, while giving it a real function name.

Reply:
Ah, it's actually one of those warnings that were not produced by my changes: `this.tt.setHtml is not a function`

Reply:
Ping @FloEdelmann . I've been doing some testing with deploy in #206 and I'm currently getting that "TypeError: this.l.ya is not a function" error and no tooltips e.g. on http://peter-ola-rdm-app.appspot.com/pid/display?manufacturer=0&pid=1 . It does work with what's on live though... http://rdm.openlighting.org/pid/display?manufacturer=0&pid=1

Reply:
Hmmm, I didn't notice a consequence of that error when testing in this PR; seems like it does matter though (it even stops showing the *upper_uid* box).

Have you already tried just replacing `this.tt.setHtml` with `this.tt.setSafeHtml` here? https://github.com/OpenLightingProject/rdm-app/blob/master/js_src/pid_display.js

It should be available (in contrast to the old `setHtml`), according to the [Closure API](https://google.github.io/closure-library/api/goog.ui.Tooltip.html).

Reply:
fixed in #209